recipe/meta.yaml

```diff
@@ -50,8 +50,7 @@ requirements:
     - python
     - setuptools
     - fastrlock >=0.3
-    - numpy >=1.16
-    - six >=1.9.0
```

Comment:
This got dropped a while ago ( https://github.com/cupy/cupy/commit/8d923719ced31fdff4a42702d395ec87a7e98045 ), but I think we missed it. Anyways we can drop it now

Reply:
Good catch! Thanks.